src/main/java/uk/gov/hmcts/reform/preapi/controllers/MediaServiceController.java

```diff
@@ -319,27 +315,41 @@ public ResponseEntity<CaptureSessionDTO> startLiveEvent(@PathVariable UUID captu
         operationId = "generateAsset",
         summary = "LEGACY - Given a source & destination, this endpoint will generate a streaming asset for a given mp4"
     )
-    @Parameter(
-        name = "code",
-        description = "Rudimentary security code to prevent unauthorised access to this endpoint",
-        schema = @Schema(implementation = String.class)
-    )
     public ResponseEntity<GenerateAssetResponseDTO> generateAsset(
```

Comment:
Might want to add a `@PreAuthorize` here to limit who can use this endpoint. I'm unsure on the requirement but I would assume only admin level users should be using this endpoint. At the very least, Level 3 users should have access to it. 

```suggestion
    @PreAuthorize("hasAnyRole('ROLE_SUPER_USER', 'ROLE_LEVEL_1')")
    public ResponseEntity<GenerateAssetResponseDTO> generateAsset(
```